lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.t

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env genome-perl
+
+# P R A G M A S ###############################################################
+use strict;
+use warnings;
+
+# M O D U L E S ###############################################################
+use Test::More;
+use Test::Number::Delta within => 1e-4;
+use Path::Class;
+use YAML::XS;
+use above 'Genome';
+
+# M A I N #####################################################################
+
+#This is a bare minimum test that just compiles Perl and the UR class.
+use_ok('Genome::Model::Tools::Vcf::EvaluateVcfs');
+
+# This directory contains the test data set (based from BIO-1176)
+my $basedir = Path::Class::Dir->new(
+    '/gscmnt/gc2801/analytics/idas',
+    'jira/BIO-1387/cle-data'
```

Comment:
Usually test data that can't be in the repository goes under the location pointed to by `$ENV{GENOME_TEST_INPUTS}`.  `Genome::Utility::Test` provides a `data_dir_ok` test that generates a path based on the module under test and checks that it exists.

Theoretically someone could then run this test externally if they were willing to supply their own dataset.